front/src/components/Login.vue

```diff
@@ -1,3 +1,29 @@
+<template>
+<div class="login_wrapper">
+    <div class="login_popup">
+        <div class="login_title">Login</div>
+        <label class="login_label" for="pseudo">pseudo</label>
+        <input v-model="pseudo" class="login_input" name="pseudo" type="text" />
+        <div class="login_submit_wrapper">
+            <button class="login_submit" @click="$emit('login', pseudo)">login</button>
+        </div>
+    </div>
+</div>
+</template>
+
```

Comment:
<!-- template for add a user to database -->

Reply:
not sure what to add here

Reply:
resolving for now